### ISSUE TITLE
No users can create Directory Facet types

### ISSUE THREAD

**Issue by j4-m** [localgovdrupal/localgov_directories#156]
After installing LocalGov Directories on a fresh install of LocalGov Drupal (2.1.6) there are no roles which have the permission to create new Directory Facet types. Consequently, only user 1 is able to create new types without further permission configuration. 

Seems like this should be streamlined. My suggestion would be to give the `editor` role as we are thinking of these types as 'content'. 

Any concerns before I go ahead and make a PR @ekes?

**Comment by ekes** [comment]
Yep looks like https://github.com/localgovdrupal/localgov_directories/blob/718701a097460e81d5c3ef06ecfa417222324c9a/localgov_directories.permissions.yml#L1 was just missed from https://github.com/localgovdrupal/localgov_directories/blob/718701a097460e81d5c3ef06ecfa417222324c9a/localgov_directories.module#L50 

They are supposed to be treated (by default) as content.

**Comment by andybroomfield** [comment]
This should be fixed in #122, but that PR is older than this issue.
We should check that it is correct on newinstalls, and also if upgrade installs can get the permissions.
(Would there be an issue with adding an update hook thats adds them if they are already assigned?)

**Comment by andybroomfield** [comment]
The localgov_editor_role has permissions.
<img width="1126" alt="Screenshot 2024-09-18 at 12 32 31 PM" src="https://github.com/user-attachments/assets/29e9cf4d-9fa1-416b-a670-c590cb792d4f">

But they can't access the menu item.
<img width="431" alt="Screenshot 2024-09-18 at 12 33 27 PM" src="https://github.com/user-attachments/assets/11d477b2-ba06-4f7c-ad94-16725263e8f9">
as they do not have access to this path
/admin/content/directories/facets/

The `Administer directory facets types` permission is required, though that has this warning.
> Warning: Give to trusted roles only; this permission has security implications. Maintain the types of directory facets entity.

We need to decide the correct behaviour here. It's going to be difficult to manage facets if they can't access them. Is there anything else the administer directory facets types gives access to? If not, maybe we can take the warning message away as although they are config entities, by default they are not exported. Else we need to have a different permission to access these.
